pnpm-lock.yaml

```diff
@@ -5194,7 +5190,7 @@ packages:
   /@prettier/sync@0.2.1(prettier@3.1.0-dev):
     resolution: {integrity: sha512-7ls1R6//+GPYD9vof1XaL5psViv83CwpdwlS8oUkWldYgbPhzZ3WgxIQMWqGyBmWPmoBfQg8C7jj7KI/ZuDHhQ==}
     peerDependencies:
-      prettier: github:cardstack/prettier#glimmer-style-tag-in-template-support
```

Comment:
hmmm, I'm not sure why pnpm decided to do this. I confirmed that the CSS in our templates still auto formats when I save in VS code. Perhaps this is no longer necessary?

Reply:
Hmm, this is puzzling to me because our https://github.com/prettier/prettier/pull/15087/ PR hasn't been merged yet to Prettier. Does formatting still work even if you `pnpm clear-caches`?